system/cmake/sdl2-config.cmake

```diff
@@ -0,0 +1,16 @@
+set(SDL2_INCLUDE_DIRS "${EMSCRIPTEN_SYSROOT}/include/SDL2")
+set(SDL2_LIBRARIES "-sUSE_SDL=2")
+
+if(NOT TARGET SDL2::SDL2)
+  add_library(SDL2::SDL2 INTERFACE IMPORTED)
```

Comment:
Could you move this to `system/lib/cmake` so we could group it with potential others?